medusa/failed_processor.py

```diff
@@ -44,8 +44,8 @@ def process(self):
             parsed = NameParser().parse(releaseName)
         except (InvalidNameException, InvalidShowException):
             self.log(logger.WARNING, u'Not enough information to parse release name into a valid show. '
-                      u'Consider adding scene exceptions or improve naming for: {release}'.format
-                      (release=releaseName))
+                     u'Consider adding scene exceptions or improve naming for: {release}'.format
+                     (release=releaseName))
```

Comment:
I would improve this saying something like this: 
 `Consider adding scene exceptions (alternative name or add name without year)`
@medariox @duramato what do you think?